yhondanilo/chess.rb

```diff
@@ -0,0 +1,46 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila, @columna=fila, columna
+    intercalar
+  end
+  def intercalar
```

Comment:
Assignment Branch Condition size for intercalar is too high. [39.82/15]<br>Use empty lines between defs.